### DISCUSSION TITLE
https://testnet10-faucet.chia.net/ does not work due to ERR_CERT_DATE_INVALID.

### DISCUSSION THREAD

**Discussion by psyto** [Chia-Network/chia-blockchain#10226] [Support]
On How to Connect to the Testnet (https://github.com/Chia-Network/chia-blockchain/wiki/How-to-connect-to-the-Testnet), I clicked the link to Chia testnet faucet (https://testnet10-faucet.chia.net/), but it did not work due to ERR_CERT_DATE_INVALID.

NET::ERR_CERT_DATE_INVALID
Subject: [testnet10-faucet.chia.net](http://testnet10-faucet.chia.net/)
Issuer: R3
Expires on: 2022/02/14
Current date: 2022/02/14

Browser: Chome 98.0.4758.80 (Official Build) (x86_64)
Machine: Mac OS X Big Sur (11.6.1)

**Comment by loppefaaret** [comment]
Thank you for the report - it should be back in working order

**Reply by psyto** [reply]
I confirmed it started working.
Thank you for your prompt action.
